services/wallet/transfer/commands_sequential.go

```diff
@@ -469,18 +464,79 @@ func loadTransfersLoop(ctx context.Context, accounts []common.Address, blockDAO
 
 					transfers = append(transfers, command.fetchedTransfers...)
 				}
-				log.Info("loadTransfers finished for account", "address", address, "in", time.Since(start), "chain", chainClient.ChainID, "transfers", len(transfers), "limit", blocksLimitPerAccount)
+				log.Debug("loadTransfers finished for account", "address", address, "in", time.Since(start), "chain", chainClient.ChainID, "transfers", len(transfers), "limit", blocksLimitPerAccount)
 			}
 
-			log.Info("loadTransfers after select", "chain", chainClient.ChainID, "address", address, "blocks.len", len(blocks))
-
 			if ok || len(blocks) == 0 ||
 				(blocksLimitPerAccount > noBlockLimit && len(blocks) >= blocksLimitPerAccount) {
-				log.Info("loadTransfers breaking loop on block limits reached or 0 blocks", "chain", chainClient.ChainID, "address", address, "limit", blocksLimitPerAccount, "blocks", len(blocks))
+				log.Debug("loadTransfers breaking loop on block limits reached or 0 blocks", "chain", chainClient.ChainID, "address", address, "limit", blocksLimitPerAccount, "blocks", len(blocks))
 				break
 			}
 		}
 	}
 
 	return nil
 }
+
+func getHeadBlockNumber(parent context.Context, chainClient *chain.ClientWithFallback) (*big.Int, error) {
+	ctx, cancel := context.WithTimeout(parent, 3*time.Second)
```

Comment:
`3` seconds timeout with a noisy internet connection seems not much. I'm wondering what are the drawbacks of one request failing often due to this 3 seconds timeout. What are the drawbacks if we set this to a 15-20 given that we expect to run these requests on a mobile network.?